tests/ship_streamer.cpp

```diff
@@ -142,7 +142,11 @@ int main(int argc, char* argv[]) {
          eosio::check(!result_document.HasParseError(),                                      "Failed to parse result JSON from abieos");
          eosio::check(result_document.IsArray(),                                             "result should have been an array (variant) but it's not");
          eosio::check(result_document.Size() == 2,                                           "result was an array but did not contain 2 items like a variant should");
-         eosio::check(std::string(result_document[0].GetString()) == "get_blocks_result_v0", "result type doesn't look like get_blocks_result_v0");
+         if( fetch_finality_data ) {
+            eosio::check(std::string(result_document[0].GetString()) == "get_blocks_result_v1", "result type doesn't look like get_blocks_result_v1");
+         } else {
+            eosio::check(std::string(result_document[0].GetString()) == "get_blocks_result_v0", "result type doesn't look like get_blocks_result_v0");
+         }
```

Comment:
Not sure we need this handling of `get_blocks_result_v0`. I think we should always send a `get_blocks_request_v1` and always expect a `get_blocks_result_v1`.

Reply:
To support Legacy test `ship_streamer_test` and newer Savanna tests, we need to handle both versions.

Reply:
No, you can just use v1 always everywhere. No need to test v0.